ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -173,6 +173,20 @@ protected override async Task ValidateAndSave()
             {
                 ModelState.AddModelError("ClinicalDetails.IsMDRTreatment", ValidationMessages.MDRCantChange);
             }
+
+            if ((ClinicalDetails.IsPostMortem ?? false))
+            {
+                if (ClinicalDetails.StartedTreatment ?? false)
+                {
+                    ModelState.AddModelError("ClinicalDetails.IsPostMortem", ValidationMessages.IsPostMortemButStartedTreatment);
+                }
+
+                if (HasNonPostMortemTreatmentEvents())
+                {
+                    ModelState.AddModelError("ClinicalDetails.IsPostMortem", ValidationMessages.IsPostMortemButTreatmentEventsDoNotMatch);
+                }
```

Comment:
This validation possibly could be added to the clinical details model if we used the trick of adding the treatment events to the model when we set the validation context. However, would understand if you don't think it's worth it.